charts/tidb-cluster/values.yaml

```diff
@@ -267,6 +267,10 @@ tidb:
   # If unset, defaults to '%' which means allow any host to connect to the TiDB.
   # permitHost: 127.0.0.1
   # initSql is the SQL statements executed after the TiDB cluster is bootstrapped.
+  # The configmap name of init sql, you can create configmap with following command:
+  # kubectl create configmap tidb-initsql --from-file=init-sql --namespace=<namespace>
```

Comment:
`--from-file=init-sql=/path/to/your/init-sql` is better, because it set the key as `init-sql` explicitly